src/Castle.Windsor/Windsor/Installer/FromAssembly.cs

```diff
@@ -115,13 +115,21 @@ public static IWindsorInstaller InDirectory(AssemblyFilter filter, InstallerFact
 		///     <c>MyApp.SuperFeatures.dll</c>, <c>mscorlib.dll</c> and <c>ThirdPartyCompany.UberControls.dll</c> the <c>MyApp.exe</c> and <c>MyApp.SuperFeatures.dll</c> will be scanned for installers, and other
 		///     assemblies will be ignored.
 		/// </remarks>
+#if FEATURE_GETCALLINGASSEMBLY
```

Comment:
I've made changes so that methods that require an `Assembly` argument are available on both, and changed the argument name to `assembly`.
How about the tests? You've mentioned cleaning them up, meaning I assume that they all will use the new methods that pass the assembly in. But then there will be no coverage for the old methods.